dotCMS/src/main/resources/mssql.sql

```diff
@@ -127,14 +127,6 @@ create table PortletPreferences (
     primary key (portletId, userId, layoutId)
 );
 
-create table Release_ (
```

Comment:
We should delete these other (non-postgres) SQL files.  We don't need to care about them.